iot_api/user_api/Error.py

```diff
@@ -35,6 +63,14 @@ class Internal(Exception):
 # Error handlers: these functions are called when an exception is raised.
 # In most cases they respond with a short message and the corresponding
 # HTML error code.
+
+@app.errorhandler(InvalidUsage)
+def handle_invalid_usage(error):
+    log.error(str(error))
```

Comment:
If the error is user faults, I think is not useful to log it. It could fill the log with error messages that are not a RoLaGuard bug.